contracts/rewards/src/contract/execute.rs

```diff
@@ -116,7 +116,7 @@ where
         contract: Addr,
         block_height: u64,
         count: Option<u64>,
-    ) -> Result<HashMap<Addr, Uint256>, ContractError> {
+    ) -> Result<HashMap<Addr, Uint128>, ContractError> {
```

Comment:
this function (and process_rewards_for_epochs) seems overly complicated. suggestions (maybe follow-up PR)
- block_height -> cur_block_height
- count -> epoch_process_limit
- instead of passing in HashMaps and continuously folding I suggest iterating over the epoch range, and returning all HashMaps or vec((Addr, Amount)) and then merging all in one go, i.e. keep the code of how to process a single epoch separate from how to bring it together for the range call

Reply:
I'd rather address this in a future PR, since it's not really related to this change set

Reply:
Well I went ahead and made the suggested renaming, since those are simple changes